frontend/libs/studio-components/src/components/StudioCodelistEditor/types/CodeListTexts.ts

```diff
@@ -0,0 +1,13 @@
+export type CodeListTexts = {
```

Comment:
nitpicking: Might `CodeListEditorTexts` be a more descriptive name for this type?

Reply:
Sure, good suggestion, I'll change it.